src/components/PrintForm/index.tsx

```diff
@@ -126,6 +126,20 @@ export const PrintForm: React.FC<PrintFormProps> = ({
     let pManagerOpts: MapFishPrintV3ManagerOpts = {
       url: ClientConfiguration.print?.url || '/print',
       map,
+      customPrintScales: map
+        ?.getView()
+        ?.getResolutions()
+        ?.map((d: number | undefined) => {
+          if (typeof d !== 'undefined') {
+            const scale = MapUtil.getScaleForResolution(d, 'm');
```

Comment:
What happens if the map units are not in meters?

Reply:
An automatic detection of map units has been added with 64fb597e54a7b786a9276f9208a7f80339fd1fd2.